electron-builder.yml

```diff
@@ -0,0 +1,16 @@
+appId: com.nervos.neuron
```

Comment:
There's still an `electron-builder.yml` file in the root folder. Was this supposed to be deleted?